tests won't start:
```
Traceback (most recent call last):
  File "/mnt/opt/Sage/sage-clang/src/bin/sage-runtests", line 9, in <module>
    sys.exit(main())
             ^^^^^^
  File "/mnt/opt/Sage/sage-clang/src/sage/doctest/__main__.py", line 192, in main
    err = DC.run()
          ^^^^^^^^
  File "/mnt/opt/Sage/sage-clang/src/sage/doctest/control.py", line 1610, in run
    self.run_doctests()
  File "/mnt/opt/Sage/sage-clang/src/sage/doctest/control.py", line 1194, in run_doctests
    self.dispatcher = DocTestDispatcher(self)
                      ^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/opt/Sage/sage-clang/src/sage/doctest/forker.py", line 1726, in __init__
    init_sage(controller)
  File "/mnt/opt/Sage/sage-clang/src/sage/doctest/forker.py", line 207, in init_sage
    controller.load_environment()
  File "/mnt/opt/Sage/sage-clang/src/sage/doctest/control.py", line 695, in load_environment
    return import_module(self.options.environment)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/mnt/opt/Sage/sage-clang/src/sage/repl/ipython_kernel/all_jupyter.py", line 6, in <module>
    from sage.all_cmdline import *
  File "/mnt/opt/Sage/sage-clang/src/sage/all_cmdline.py", line 17, in <module>
    from sage.all import *
  File "/mnt/opt/Sage/sage-clang/src/sage/all.py", line 86, in <module>
    from sage.rings.all import *
  File "/mnt/opt/Sage/sage-clang/src/sage/rings/all.py", line 57, in <module>
    from sage.rings.number_field.all import *
  File "/mnt/opt/Sage/sage-clang/src/sage/rings/number_field/all.py", line 2, in <module>
    from sage.rings.number_field.number_field import (NumberField, NumberFieldTower, CyclotomicField, QuadraticField,
  File "/mnt/opt/Sage/sage-clang/src/sage/rings/number_field/number_field.py", line 87, in <module>
    import sage.rings.real_mpfi
  File "real_mpfi.pyx", line 1, in init sage.rings.real_mpfi
  File "mpfi.pyx", line 1, in init sage.rings.convert.mpfi
ImportError: /mnt/opt/Sage/sage-clang/build/cp312/src/sage/rings/complex_interval.cpython-312-x86_64-linux-gnu.so: undefined symbol: fmpz_tdiv_q_2exp
```

The latter symbol is from `libflint`, but `complex_interval.cpython-312-x86_64-linux-gnu.so` is not linked to `libflint`

--- a/src/meson.build
+++ b/src/meson.build
@@ -86,7 +86,7 @@ gsl = dependency(
 )
 gd = cc.find_library('gd')
 # Only some platforms have a standalone math library (https://mesonbuild.com/howtox.html#add-math-library-lm-portably)
-m = cc.find_library('m', required : false)
+m = cc.find_library('m', required: false)
 m4ri = cc.find_library('m4ri')
 m4rie = cc.find_library('m4rie')
 mtx = cc.find_library('mtx', required: false, disabler: true)

--- a/src/sage/groups/matrix_gps/meson.build
+++ b/src/sage/groups/matrix_gps/meson.build
@@ -36,7 +36,7 @@ foreach name, pyx : extension_data
   deps = [py_dep, gmp]
   if name == 'group_element_gap'
     deps += [gap]
-  endif    
+  endif
 
   py.extension_module(
     name,

--- a/src/sage/matrix/meson.build
+++ b/src/sage/matrix/meson.build
@@ -96,11 +96,7 @@ extension_data = {
 }
 
 foreach name, pyx : extension_data
-  dependencies = [
-    py_dep,
-    cysignals,
-    gmp,
-  ]
+  dependencies = [py_dep, cysignals, gmp]
   if name == 'matrix_gfpn_dense'
     dependencies += [mtx, meataxe]
   elif name == 'matrix_gap'

--- a/src/sage/rings/finite_rings/meson.build
+++ b/src/sage/rings/finite_rings/meson.build
@@ -41,7 +41,10 @@ extension_data = {
 foreach name, pyx : extension_data
   deps = [py_dep, cysignals, gmp]
   if name == 'element_pari_ffelt'
-    deps += [cypari2]
+    deps += [
+      cypari2,
+      pari,  # Runtime dependency
+    ]
   elif name == 'residue_field_pari_ffelt'
     deps += [cypari2]
   endif

--- a/src/sage/rings/meson.build
+++ b/src/sage/rings/meson.build
@@ -124,13 +124,20 @@ extension_data = {
 foreach name, pyx : extension_data
   deps = [py_dep, cysignals, gmp]
   if name == 'complex_arb'
-    deps += [flint, mpfi]
+    deps += [
+      flint,
+      mpfi,
+      gsl,  # Runtime dependency
+    ]
   elif name == 'complex_conversion'
     deps += [gsl, mpfr]
   elif name == 'complex_double'
     deps += [gmpy2, gsl]
   elif name == 'complex_interval'
-    deps += [mpfi]
+    deps += [
+      mpfi,
+      flint,  # Runtime dependency
+    ]
   elif name == 'complex_mpc'
     deps += [gmpy2, mpfr, mpc]
   elif name == 'complex_mpfr'

--- a/src/sage/rings/polynomial/meson.build
+++ b/src/sage/rings/polynomial/meson.build
@@ -92,11 +92,7 @@ extension_data = {
 }
 
 foreach name, pyx : extension_data
-  deps = [
-      py_dep,
-      cysignals,
-      gmp,
-    ]
+  deps = [py_dep, cysignals, gmp]
   if name == 'evaluation_flint'
     deps += [flint, mpfi]
   elif name == 'polynomial_complex_arb'

--- a/src/sage/symbolic/meson.build
+++ b/src/sage/symbolic/meson.build
@@ -47,11 +47,7 @@ foreach name, pyx : extension_data
       inc_pynac,
       inc_rings,
     ],
-    dependencies: [
-      py_dep,
-      cysignals,
-      gmp,
-    ],
+    dependencies: [py_dep, cysignals, gmp],
   )
 endforeach
 
@@ -128,13 +124,7 @@ foreach name, pyx : extension_data_cpp
       inc_rings,
       include_directories('../libs/gmp'),
     ],
-    dependencies: [
-      py_dep,
-      cysignals,
-      gmp,
-      gsl,
-      singular,
-    ],
+    dependencies: [py_dep, cysignals, gmp, gsl, singular],
   )
 endforeach
 

